README.md

```diff
@@ -1,91 +1,120 @@
-
-**[React-Redux-Capstone]** is an HTML, CSS and Javascript project
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 capstone ](#-capstone-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [ Video Demo ](#video-demo)
+    - [ Live Link ](#live-link)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖React-Redux Capstone Project<a name="about-project"></a>
+
+This is the capstone project for react. It consumes an API, uses Redux as a state management tool and React for the UI frontend.
 
 ## 🛠 Built With <a name="built-with"></a>
+- React
+- Redux
+- API
 
 ### Tech Stack <a name="tech-stack"></a>
 
-- React.js
-- Redux
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">React JS</a></li>
+  </ul>
+</details>
 
 ### Key Features <a name="key-features"></a>
-- **Basic maths Calculations**
-- **Well structured UI**
 
-## 💻 Getting Started <a name="getting-started"></a>
+- **Code Quality**
+- **SPAs**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-To get a local copy up and running, follow these steps.
+### Video Demo <a name="video-demo"></a>
+
+[Video Link](https://www.loom.com/share/d1bb499c11b240ea99069a1166b2e901?sid=66b92fdb-1145-423e-899a-199ed82382b3)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Link <a name="live-link"></a>
+
+[Live Link](https://ngalamac.github.io/react-redux-capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
 
 ### Prerequisites
 
 In order to run this project you need:
 
-- Visual Studio Code
-- Web browser such as chrome
+- You need to have NodeJS installed
+- A Web Browser (Google Chrome, Firefox, etc)
+- A Code Editor (Notepad++, VSCode, etc)
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-```sh
-  cd react-redux-capstone
-  git clone https://github.com/ngalamac/react-redux-capstone.git
+```
+git clone your link repo https://github.com/ngalamac/react-redux-capstone.git
+
+cd react-redux-capstone
 ```
 
 ### Install
 
 Install this project with:
 
-```sh
-  cd react-redux-capstone
-  npm install
+```
+npm install
 ```
 
 ### Usage
 
-To run the project, start the html file on live server or run normally from local directory
+To run the project, execute the following command:
+
+``npm start``
 
 ### Run tests
 
 To run tests, run the following command:
```

Comment:
- [ ] 👋 Hi there! It looks like your README is off to a great start! To make it even more helpful, consider including the command for running Jest tests in your README. This can help others easily test your project.